modules/main.js

```diff
@@ -203,12 +203,24 @@ function processAction(aWindow, aAction, aRootElement)
   var value = actions[2];
   switch (action) {
   case 'accept':
-    commonDialog.ui.button0.click();
     log("accept");
+    if (commonDialog) {
+      commonDialog.ui.button0.click();
+    } else if (typeof root.acceptDialog == "function") {
+      root.acceptDialog();
+    } else {
+      Cu.reportError(new Error("We don't know how to accept "+root));
```

Comment:
How about using `root.id` instead of `root`? I like element ID such as `SanitizeDialog` better than `[object XULElement]`.